pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -26,6 +26,10 @@ buildPythonPackage rec {
     hash = "sha256-UTeZXlNuSWDOcBRVbUUWJ3mQnZZQr17aTw6rRA5DRNI=";
   };
 
+  patches = [
+    ./coerce-numpy-to-int.patch
```

Comment:
Please give a short comment what it does and possibly link to the upstream issue, so that we known when it can be removed :)

Reply:
Issue created. https://github.com/pyscf/pyscf/issues/2878